api/src/pcapi/core/users/factories.py

```diff
@@ -361,6 +361,38 @@ class Params:
     )
 
 
+class EligibleUnderageCompletedAllStepsFactory(EligibleUnderageFactory):
+    @factory.post_generation
+    def beneficiaryFraudChecks(  # pylint: disable=no-self-argument
+        obj,
+        create: bool,
+        extracted: fraud_models.BeneficiaryFraudCheck | None,
+        **kwargs: typing.Any,
+    ) -> fraud_models.BeneficiaryFraudCheck | None:
+        import pcapi.core.fraud.factories as fraud_factories
+
+        birth_date = datetime.utcnow() - relativedelta(years=obj.age, months=5)
+        fraud_factories.BeneficiaryFraudCheckFactory(
+            user=obj,
+            type=fraud_models.FraudCheckType.EDUCONNECT,
+            status=fraud_models.FraudCheckStatus.OK,
+            eligibilityType=models.EligibilityType.UNDERAGE,
+            resultContent=fraud_factories.EduconnectContentFactory(
+                first_name=obj.firstName,
+                last_name=obj.lastName,
+                birth_date=birth_date,
+            ),
+        )
+        fraud_factories.BeneficiaryFraudCheckFactory(
+            user=obj,
+            type=fraud_models.FraudCheckType.HONOR_STATEMENT,
+            status=fraud_models.FraudCheckStatus.OK,
+            eligibilityType=models.EligibilityType.UNDERAGE,
+        )
+        fraud_factories.ProfileCompletionFraudCheckFactory(user=obj, eligibilityType=models.EligibilityType.UNDERAGE)
+        return None
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏Pas besoin de `return None`